openlibrary/plugins/upstream/models.py

```diff
@@ -413,14 +413,19 @@ def wp_citation_fields(self):
         result = {
             "title": self.works[0].title.replace("[", "&#91").replace("]", "&#93"),
```

Comment:
this line should be:
`"title": self.title.replace("[", "&#91").replace("]", "&#93"),`

We want the title of `self` , which is`class Edition`, and the replace is specified by the Wikipedia citation template for its `title:`   

Apologies for any confusion from  the double usage of edition.
Apart from this one line, everything else is good as is :+1: